openshift-ansible.spec

```diff
@@ -85,8 +85,7 @@ touch %{buildroot}%{_datadir}/ansible/%{name}/roles/contiv/.empty_dir
 %doc README*
 %license LICENSE
 %dir %{_datadir}/ansible/%{name}
-%{_datadir}/ansible/%{name}/inventory/dynamic
-%ghost %{_datadir}/ansible/%{name}/playbooks/common/openshift-master/library.rpmmoved
+%{_datadir}/ansible/%{name}/inventory
```

Comment:
It seems strange to me we need to call out this one sub directory and not any of the others.

Reply:
What other directories do you have in mind?

Reply:
nvm, I see the rest are covered in the subpackages.